audio-worklet/basic/js/one-pole.js

```diff
@@ -1,28 +1,44 @@
 /**
+ * A simple One pole filter.
+ * 
  * @class OnePole
  * @extends AudioWorkletProcessor
- *
- * Implements a simple OnePole filter. 
  */
 class OnePole extends AudioWorkletProcessor {
+  static get parameterDescriptors() {
+    return [{
+      name: 'frequency',
+      defaultValue: 250,
+      minValue: 0
```

Comment:
Since we chatted about this, maybe add comment that maxValue is defaulted on purpose because we can't set it to the (unknown) Nyquist frequency?

Reply:
Done.